client/extensions/woocommerce/components/list/list-row/index.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
My vote is to not check in unused code - maybe throw it in a public gist (and link to it from here) for safekeeping?